src/partials/global-nav.html

```diff
@@ -17,7 +17,9 @@
         {{#each language.nav}}
         {{> navlist-item item=.}}
         {{/each}}
+        
         <li class="nav-item lang">
+          <a href="https://www.google.com/search?client=firefox-b-d&q=world+corona+statistic" style="color:rgb(51, 51, 146);text-decoration: underline;"> Corona Virus Statistic</a>
```

Comment:
Focusing on the link target and not the idea as such:
- please remove add least the `client=firefox-b-d` - this does not belong here
- why link to google search, when google search is sourcing its data from wikipedia?
![Bildschirmfoto von 2020-08-16 16-54-26](https://user-images.githubusercontent.com/24962768/90337203-3ea72b00-dfe1-11ea-8a23-ebad833e587d.png)
- why not link to the RKI? (e.g. [Fallzahlen](https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Fallzahlen.htmll) or [COVID-19-Dashboard](https://corona.rki.de/))

As you can see in your own screenshot:
- the color you are setting has little contrast to the background